omniduct/filesystems/s3.py

```diff
@@ -189,11 +191,73 @@ def _dir(self, path):
     #         if pattern is None or pattern.match(k.key[len(path_prefix):]):
     #             yield k.key
 
+    def _format_s3_path(self, path):
+        return 's3:' + path[4:] if path.startswith('s3n:') or path.startswith('s3a:') else path
+
+    def _is_s3(self, path):
```

Comment:
The paths passed onto these methods preceded by an underscore (_) are already sanitised (with paths relative to the root of the bucket).

Reply:
So, can I just remove these two methods _format_s3_path and _is_s3?

Reply:
Yep :).

Reply:
Done